web/src/lib/components/admin-page/settings/setting-accordion.svelte

```diff
@@ -8,7 +8,7 @@
 </script>
 
 <div class="border-b-[1px] border-gray-200 py-4 dark:border-gray-700">
-  <div class="flex place-items-center justify-between">
+  <button on:click={toggle} class="flex cursor-pointer place-items-center justify-between text-left">
```

Comment:
Button doesn't need cursor-pointer btw

Reply:
Ah yes indeed.. It was a div in my initial commit, I forgot to remove it afterwards.

Should I make a new PR?

Reply:
Not a big deal.. would not hurt to fix it either.

Reply:
It seems like the changes after my review also did something that caused the arrow button to move to the left. Wouldn't hurt to fix that either.
![image](https://github.com/immich-app/immich/assets/11602424/4a840921-1867-483c-b537-f866c848c9a3)

You mentioned you had trouble with your dev env - if you want some help fixing that don't hesitate to hop into Discord.

Reply:
You can try this too

https://immich.app/docs/developer/setup#connect-to-a-remote-backend

Reply:
Sorry about the issue caused by my commit. I've fixed it here if you'd like to take a look : #4129 

jrasm the remote backend works much better than my local env, thanks. :) I'll post a message on discord to get help if i get stuck again !

Reply:
Thank you!